### PR TITLE
Add Laser Kombat

### PR DESCRIPTION
**Pull request by sharkwouter**
Laser Kombat is a very old Laser Kombat clone which I ported to SDL2.

### CODE REVIEW

File: games/l.yaml [hunk 1]
```diff
@@ -61,6 +61,28 @@
   - 'https://ktpanda.org/software/lab3d_sdl/images/shot03.jpeg'
   - 'https://ktpanda.org/software/lab3d_sdl/images/shot04.jpeg'
 
+- name: Laser Kombat
+  originals:
+  - LaserTank
+  type: clone
+  repo: 'https://github.com/sharkwouter/laserkombat'
+  url: 'https://github.com/sharkwouter/laserkombat'
```
Comment (cxong) [inline games/l.yaml:69]:
```suggestion
```
don't need this as it duplicates repo

File: games/l.yaml [hunk 2]
```diff
@@ -61,6 +61,28 @@
...
+  development: active
```
Comment (cxong) [inline games/l.yaml:70]:
```suggestion
  development: sporadic
```

**Review by cxong — CHANGES_REQUESTED** [review]

**Review by cxong — APPROVED** [review]

File: games/l.yaml [hunk 3]
```diff
@@ -61,6 +61,27 @@
   - 'https://ktpanda.org/software/lab3d_sdl/images/shot03.jpeg'
   - 'https://ktpanda.org/software/lab3d_sdl/images/shot04.jpeg'
 
+- name: Laser Kombat
```
Comment (cxong) [inline games/l.yaml:64]: please sort alphabetically (after ladder, before LastNinja)

Comment (sharkwouter) [reply games/l.yaml]: Oh, I missed ladder there. Fixed now, thanks!

**Pull request merged**